js/react/components/auth/login/LoginWithEmail.jsx

```diff
@@ -1,72 +1,141 @@
-								className="form-control rounded-pill"
-								id="password"
-								placeholder="Password"
-							/>
-							<label htmlFor="password">Password</label>
-						</div>
-						<button className="w-100 mt-5 mb-3 button-modal-secondary" type="submit">
-							Log in
-						</button>
-						<div className="w-100 d-flex justify-content-between align-items-start align-items-sm-center flex-column flex-sm-row gap-2 gap-sm-0">
-							<div className="form-check">
+		<>
+			{error !== '' ? <CustomToast type="error" title={error} /> : null}
+			<Modal
+				ref={modalRef}
+				title={<h1 className="cool-font text-secondary">Log in with email</h1>}
+				body={
+					<div className="d-flex flex-column align-items-center justify-content-center gap-5">
+						<form action="" method="post" className="w-100">
+							<div className="form-floating mb-3">
 								<input
-									className="form-check-input"
-									type="checkbox"
-									value=""
-									id="rememberMe"
+									type="email"
+									className="form-control rounded-pill"
+									id="email"
+									placeholder="Email"
+									value={email}
+									onChange={(e) => setEmail(e.target.value)}
+									required
 								/>
-								<label className="form-check-label" htmlFor="rememberMe">
-									Remember me
-								</label>
+								<label htmlFor="email">Email</label>
 							</div>
-							<a href="/profile/password/reset" className="text-secondary">
-								Forgot password?
-							</a>
+							<div className="form-floating position-relative">
+								<input
+									type={hidePassword ? 'password' : 'text'}
+									className="form-control rounded-pill"
+									id="password"
+									placeholder="Password"
+									value={password}
+									onChange={(e) => setPassword(e.target.value)}
+									required
+								/>
+								<label htmlFor="password">Password</label>
+								<i
+									className={
+										'show-password fas ' +
+										(hidePassword ? 'fa-eye' : 'fa-eye-slash')
+									}
+									role="button"
+									onClick={togglePasswordVisibility}
+								></i>
+							</div>
+							<button
+								className="w-100 mt-5 mb-3 button-modal-secondary"
+								type="submit"
+								onClick={loginWithEmailHandler}
+							>
+								Log in
+							</button>
+							<div className="w-100 d-flex justify-content-between align-items-start align-items-sm-center flex-column flex-sm-row gap-2 gap-sm-0">
+								<div className="form-check">
+									<input
+										className="form-check-input"
+										type="checkbox"
+										value=""
+										id="rememberMe"
+									/>
+									<label className="form-check-label" htmlFor="rememberMe">
+										Remember me
+									</label>
+								</div>
+								<a href="/profile/password/reset" className="text-secondary">
+									Forgot password?
+								</a>
+							</div>
+						</form>
+					</div>
+				}
+				footer={<p>Agency partner users must sign in with work email</p>}
```

Comment:
For login page, we will be consistent with "log in", so please change the text to "Agency partner users must log in with work email." I've already changed this in my previous PR, but somehow the change is lost.